modules/smtp/modules.php

```diff
@@ -939,6 +946,132 @@ protected function output() {
             ' value="1" id="enable_compose_delivery_receipt" name="enable_compose_delivery_receipt" />'.$reset.'</td></tr>';
     }
 }
+            
+/**
+ * Send scheduled messages
+ * @subpackage smtp/handler
+ */
+class Hm_Handler_send_scheduled_messages extends Hm_Handler_Module {
+    /**
+     * Send delayed messages
+     * This should use cron
+     */
+    public function process() {
+        if (!($this->module_is_supported('imap') || $this->module_is_supported('profiles'))) {
+            return;
+        }
+        $servers = Hm_IMAP_List::dump();
+        $scheduled_msg_count = 0;
+            foreach (array_keys($servers) as $server_id) {
```

Comment:
The whole indentation of this block is wrong and hard to read.

Reply:
I aligned for better readability, which allows you to quickly understand the hierarchy and logic of the code